go.mod

```diff
@@ -60,6 +60,7 @@ require (
 	github.com/stretchr/testify v1.8.3
 	github.com/testcontainers/testcontainers-go v0.19.0
 	github.com/xuri/excelize/v2 v2.7.1
+	github.com/zeebo/assert v1.3.0
```

Comment:
Is this needed – we use `github.com/stretchr/testify` for testing assertions

Reply:
go land added that will remove